test/e2e/handler/node_selector_test.go

```diff
@@ -25,8 +25,8 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		AfterEach(func() {
-			By(fmt.Sprintf("Deleteting linux bridge %s", bridge1))
-			setDesiredStateWithPolicy(bridge1, linuxBrAbsent(bridge1))
+			By(fmt.Sprintf("Deleteting linux bridge %s at all nodes", bridge1))
+			setDesiredStateWithPolicyWithoutNodeSelector(bridge1, linuxBrAbsent(bridge1))
```

Comment:
Why do we need this? We configured it only on those matching a label. If it is to remove the nested policy without a label, we should create its own AfterEach in its respective context. As a reader of this Context and its It, this cleanup confuses me.

Reply:
Explained at the commit messages.

Reply:
I think we should keep this cleanup and create a new one in the nested. It cleans up the nested policy, it is not needed for this level.

Reply:
It does the cleanup from the stuff populated at `BeforeEach` so this level make sense.